test fails on python3

I'm not sure whether the package is intended to run on python3, but the PyPI entry at least does not specify otherwise. When running the tests on python3, I get an error
````
=========================== FAILURES ===========================
______________________ test_get_entry_map ______________________

bkend = <reentry.jsonbackend.JsonBackend object at 0x10528e2e8>

    def test_get_entry_map(bkend):
        """
        The map for distA in the fixture should contain
        a group and an entry point
        """
        from .. import manager
        manager.bkend = bkend
>       epmap = manager.get_entry_map('distA')

reentry/tests/test_manager.py:21:
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _
reentry/manager.py:29: in get_entry_map
    return bkend.get_map(dist=distname, group=group)
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _

self = <reentry.jsonbackend.JsonBackend object at 0x10528e2e8>
dist = 'distA', group = ['groupA'], name = None

    def get_map(self, dist=None, group=None, name=None):
        """
            see BackendInterface docs
            """
        import re
        from collections import Sequence
        from reentry.entrypoint import EntryPoint
        '''sanitize dist kwarg'''
        if dist not in self.epmap:
            raise ValueError("The {} distribution was not found.".format(dist))

        '''sanitize groups kwarg'''
        if group is None:
            group = self.get_group_names()
>       if not isinstance(group, Sequence) or isinstance(group, (str, unicode)):
E       NameError: name 'unicode' is not defined

reentry/jsonbackend.py:179: NameError
=================== 1 failed in 0.09 seconds ===================
TESTS FAILED: reentry-1.0.2-py35_0.tar.bz2
```

--- a/reentry/abcbackend.py
+++ b/reentry/abcbackend.py
@@ -66,7 +66,6 @@ class BackendInterface(object):
         take a distribution's project name, add the distribution
         """
         if entry_point_map:
-            print distname, entry_point_map
             self.write_dist_map(distname=distname, entry_point_map=entry_point_map)
         else:
             dist = self.pr_dist_from_name(distname)

--- a/reentry/jsonbackend.py
+++ b/reentry/jsonbackend.py
@@ -285,6 +285,6 @@ def _listify(sequence_or_name):
     from collections import Sequence
     if sequence_or_name is None:
         return None
-    elif not isinstance(sequence_or_name, Sequence) or isinstance(sequence_or_name, (str, unicode)):
+    elif not isinstance(sequence_or_name, Sequence) or isinstance(sequence_or_name, six.string_types):
         return [sequence_or_name]
     return sequence_or_name

--- a/reentry/manager.py
+++ b/reentry/manager.py
@@ -1,5 +1,8 @@
 # -*- coding: utf8 -*-
 """Manager for entry point based plugins. Main client facing API"""
+
+import six
+
 from reentry.jsonbackend import JsonBackend
 
 DEFAULT_BACKEND = JsonBackend()
@@ -58,9 +61,9 @@ class PluginManager(object):
             dist = dists[0]
             emap = dist.get_entry_map()
             if groups:
-                dmap = {k: v for k, v in emap.iteritems() if k in groups}
+                dmap = {k: v for k, v in six.iteritems(emap) if k in groups}
             elif group_re:
-                dmap = {k: v for k, v in emap.iteritems() if group_re.match(k)}
+                dmap = {k: v for k, v in six.iteritems(emap) if group_re.match(k)}
             else:
                 dmap = None
             dname = dist.project_name

--- a/reentry/manager.py
+++ b/reentry/manager.py
@@ -64,7 +64,6 @@ class PluginManager(object):
             else:
                 dmap = None
             dname = dist.project_name
-            print dname, dmap
             self._backend.write_dist(dname, entry_point_map=dmap)
 
     def unregister(self, distname):

